Fruits.txt

```diff
@@ -6,4 +6,5 @@ Starfruit
 Jackfruits
 Watermelons
 Oranges
-Pineapple
\ No newline at end of file
+Pineapple
+Pear
```

Comment:
Will consider the Pineapple removal and addition for now. 
Approved.

Reply:
Thanks. Please approve and merge.